packages/widget/src/hooks/useAddressActivity.ts

```diff
@@ -1,33 +1,48 @@
-import type { Address } from 'viem'
-import { isAddress } from 'viem'
-import { useTransactionCount } from 'wagmi'
+import { ChainType } from '@lifi/sdk'
+import { useEthereumContext } from '@lifi/widget-provider'
+import { useQuery } from '@tanstack/react-query'
+import { useWidgetConfig } from '../providers/WidgetProvider/WidgetProvider.js'
 import { useFieldValues } from '../stores/form/useFieldValues.js'
+import { getQueryKey } from '../utils/queries.js'
+import { useAvailableChains } from './useAvailableChains.js'
 
 interface AddressActivity {
   hasActivity: boolean
   isLoading: boolean
   isFetched: boolean
   toAddress: string | undefined
 }
-
 export const useAddressActivity = (chainId?: number): AddressActivity => {
+  const { keyPrefix } = useWidgetConfig()
```

Comment:
Why do we need `keyPrefix` here?

Reply:
There are some queries that should have keyPrefix when there are two widgets on the page (Jumper does this), but for some of them it's okay to be shared, that's why those didn't have it.